cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -130,6 +130,12 @@ constexpr bool type_id_matches_device_storage_type(type_id id)
          (id == type_id::DECIMAL128 && std::is_same_v<T, __int128_t>) || id == type_to_id<T>();
 }
 
+template <typename T>
+constexpr inline type_id type_to_id<Type>()
```

Comment:
This just needs some doxygen to pass the style-checker.

Reply:
Probably just copy the one from `base_type_to_id`

Reply:
Done.